src/components/product/ProductGrid/ProductGrid.tsx

```diff
@@ -12,7 +13,7 @@ interface Props {
 function ProductGrid({ products, page, pageSize }: Props) {
   return (
     <ProductGridSkeleton loading={products.length === 0}>
-      <ul className="product-grid">
+      <ul className={styles.productGrid}>
```

Comment:
Noticed something is off in this grid, both on mobile and desktop (see https://preview-54--gatsby.preview.vtex.app/office):
![CleanShot 2022-05-19 at 21 05 08](https://user-images.githubusercontent.com/381395/169423665-7bd40ed8-f926-4053-b2f2-7edd618a27c3.png)

I think we're missing this attribute here:
```suggestion
      <ul data-fs-product-grid className={styles.productGrid}>
```

And in `product-grid-module.scss`:

```diff
- .product-grid {
+ .product-grid[data-fs-product-grid] {
```

Not sure.

Reply:
I also noticed this issue. But I found that running the project locally this isn't happening. 

|Preview|
|-|
![preview](https://user-images.githubusercontent.com/3356699/169426611-ac37f818-df61-4847-a0ff-2df6e17876b8.png)
 
|Localhost|
|-|
![localhost](https://user-images.githubusercontent.com/3356699/169426455-48544c31-df44-4266-b1f7-536db7911184.png)

I think we have faced a similar issue before.
In production, we are enabling the CSS code-splitting but locally remain a single CSS file (commons.css)
I'm not sure what would be the best approach for this case. We can fix this individually, but I don't think it's the best approach since we will be using CSS modules.
@eduardoformiga any thoughts?  (Should we add that plugin again 👀 )

Reply:
I think I have fixed it by using CSS Modules to import the `ProductGallery` section related CSS file. See https://github.com/vtex-sites/gatsby.store/pull/54/commits/014841949781e0806a1e8e17fd6255ff8d82d295

Reply:
That's true, @hellofanny .  We deal with it on https://github.com/vtex-sites/base.store/pull/480

Unfortunately, we have different approaches from dev and prod.